akka-http-core/src/main/scala/akka/http/impl/util/package.scala

```diff
@@ -127,6 +127,66 @@ package object util {
 
 package util {
 
+  import akka.http.scaladsl.model.{ ContentType, HttpEntity }
```

Comment:
Should I refactor http.impl.util into an object instead? I think there are a couple of package objects in http, so those should be changed as well. I think that should be another ticket and PR though (these are used all around the place).